source/grid/grid_tools.cc

```diff
@@ -556,7 +556,7 @@ namespace GridTools
     class ShiftPoint
```

Comment:
I profess no preference one way or the other. I'll change it to `Shift`, and do the same transformation for `ScalePoint` a few lines below.